### PR TITLE
[docs] Documents in grin repo had translated in Korean. 

### PR DESCRIPTION
**Pull request by cmcm2222**
I translated some  grin repo's documents in Korean. 
list below.
 
- stratum_KR.md
- table_of_contents_KR.md
- intro_KR.md

### CODE REVIEW

**Comment by cmcm2222** [comment]
I don't know why my integration was failed, anybody have idea about it?

**Comment by hashmap** [comment]
@cmcm2222 I restarted the build, seems like a Travis glitch

File: doc/table_of_contents_KR.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# 문서 목록
+
+## Grin 에 대한 설명들
+
+- [intro](intro.md) - Grin 에 대한 기술적인 소개
```
Comment (ignopeverell) [inline doc/table_of_contents_KR.md]: Why not linking directly to `intro_KR.md`?

File: doc/code_structure.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # Grin code structure
 
 Grin is built in [Rust](https://www.rust-lang.org/), a memory safe, compiled language. Performance critical parts like the Cuckoo mining algorithm are built as plugins, making it easy to swap between algorithm implementations for various hardware. Grin comes with CPU and experimental GPU support.
-
```
Comment (ignopeverell) [inline doc/code_structure.md]: This line break seems good to have.

**Comment by russwentz** [comment]
there's quite a lot of commits in this PR, i think it should be squashed before merging.

File: doc/code_structure.md (hunk 2 above)
Comment (cmcm2222) [reply doc/code_structure.md]: thanks. 
I will. :)

File: doc/table_of_contents_KR.md (hunk 1 above)
Comment (cmcm2222) [reply doc/table_of_contents_KR.md]: Oh. My miss. 
Thank you :)

File: doc/code_structure.md (hunk 2 above)
Comment (cmcm2222) [reply doc/code_structure.md]: @yeastplume ping. :)

**Pull request merged**